.travis.yml

```diff
@@ -18,6 +17,7 @@ env:
   - TASK=cpp_test
   # run tests/python
   - TASK=python_test
+  #- TASK=r_test
```

Comment:
Oh BTW, I forgot to uncomment the slack notification in Travis file. Did Travis send a lot of messages to your channel before I commented out? (Hope didn't bother you guys a lot...)